docs/reference/compose.md

```diff
@@ -44,7 +44,6 @@ Define and run multi-container applications with Docker
 | [`wait`](compose_wait.md)       | Block until containers of all (or specified) services stop.                             |
 | [`watch`](compose_watch.md)     | Watch build context for service and rebuild/refresh containers when files are updated   |
 
-
```

Comment:
section between `MARKER_GEN` tags **MUST** be left unchanged (or fixed must apply to docs generator)